ivy/ivy.xml

```diff
@@ -76,10 +76,10 @@
     <dependency org="com.google.guava" name="guava" rev="11.0.2" />
     <dependency org="com.google.code.crawler-commons" name="crawler-commons" rev="0.5" />
     
-    <dependency org="org.restlet.jse" name="org.restlet" rev="2.3.7" conf="*->default" />
-    <dependency org="org.restlet.jse" name="org.restlet.ext.jackson" rev="2.3.7" conf="*->default" />
-    <dependency org="org.restlet.jse" name="org.restlet.ext.jaxrs" rev="2.3.7" conf="*->default" />
-    <dependency org="org.restlet.jee" name="org.restlet.ext.crypto" rev="2.3.7" conf="*->default" />
+    <dependency org="org.restlet.jse" name="org.restlet" rev="2.2.3" conf="*->default" />
```

Comment:
I had to solve very challenging tasks to solve authorization at NutchServer. When you construct dependencies of Nutch you will see that jersey 1.9 is used and it is to dangerous to upgrade it to newer major version: 2.x.

When you use jersey 2.x you have an ability to use @RolesAllowed, @Priority or ContainerRequestFilter etc. but you can't do it with current dependency complexity at Nutch. 

You can use filters with restlet 2.3.7 for authorization but I couldn't not use restlet 2.3.7 (I've reverted the version to 2.2.3) because restlet 2.3.7 forces you to use jersey 2.x when you want to use such filters.